Library/Homebrew/api/formula.rb

```diff
@@ -33,7 +33,7 @@ def all_formulae
             if cached_formula_json_file.exist?
```

Comment:
```suggestion
            if cached_formula_json_file.exist? && !cached_formula_json_file.empty?
```
It makes more sense to have this condition up here, no?

Reply:
Yes, this makes more sense--fixed.